staging/src/k8s.io/apiserver/pkg/endpoints/handlers/delete.go

```diff
@@ -224,6 +225,7 @@ func DeleteCollection(r rest.CollectionDeleter, checkBody bool, scope *RequestSc
 		}
 
 		options := &metav1.DeleteOptions{}
+		options.TypeMeta.SetGroupVersionKind(metav1.SchemeGroupVersion.WithKind("DeleteOptions"))
```

Comment:
dup of L269

Reply:
it seems that we didn't hook up deleteOptions in API installer for deletecollection endpoints (and OpenAPI as a result). I will send a fix

Reply:
Thanks, I've remove the typemeta set on L228.